ANTLR/src/Compiler/Main.java

```diff
@@ -27,6 +27,7 @@ public static void main(String[] args) throws IOException {
             runCompiler(args);
         }
         catch (Exception e){
+            System.out.println(e.getMessage());
```

Comment:
This is for debugging purposes only. Should probably be deleted before merging with master.